_layouts/tutorial.html

```diff
@@ -9,6 +9,9 @@
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <meta name="description" content="Get started developing Apache Kafka® applications with this step-by-step tutorial to learn how to {{ site.data.tutorials[page.static_data].meta-description }} using {{ site.data.stacks[page.stack].pretty }}.">
+  {% unless page.stack == site.data.tutorials[page.static_data].canonical or page.stack == 'ksql' %}
```

Comment:
if `canonical` is not defined for the page and the stack is not `ksql`, the canonical tag would still be rendered, and the href value below would be something like `.../false.html` or `../undefined.html`

Reply:
@colinhicks I see that now too, thanks for catching it.  I will correct

Reply:
Corrected in latest commit